### PR TITLE
docs: Improve Quick start instructions

### PR DESCRIPTION
**Pull request by waldyrious**
**Summary**

- Slightly tweak Python server instructions to be more generic
- Clarify the section about customizing the main container

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Repo settings
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

### CODE REVIEW

**Review by sy-records — APPROVED** [review]

File: docs/quickstart.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ If you want, you can show a loading dialog before docsify starts to render your
 <div id="app">Please wait...</div>
 ```
 
-You should set the `data-app` attribute if you changed `el`:
+By default, the `id` attribute of the main container is `app`. If you want to use a different id, you must also set the `data-app` attribute, and [configure](configuration.md#el) the `el` parameter accordingly:
```
Comment (trusktr) [inline docs/quickstart.md:100]: A bit weird that we need to set `data-app` like this. Let's fix in v5.

/wish-there-was-a-bot-command-to-open-an-new-issue-right-from-here

**Review by trusktr — APPROVED** [review]
Thanks for this!

File: docs/quickstart.md (hunk 1 above)
Comment (sy-records) [reply docs/quickstart.md:100]: I added it manually https://github.com/docsifyjs/docsify/projects/2#card-51972364

**Comment by sy-records** [comment]
Sorry, The branch is wrong, We didn't notice... @trusktr 😅

**Comment by waldyrious** [comment]
> Sorry, The branch is wrong, We didn't notice... @trusktr 😅

I see that `develop` is already the default branch on GitHub, so this shouldn't happen from PRs initiated from the GitHub interface. However, in this case I clicked the "Edit document" link at https://docsify.js.org/#/quickstart. It seems like the edit code needs to be fixed to target the correct branch. 🙂

I noticed the `master` branch being hardcoded in several places:

- https://github.com/docsifyjs/docsify/blob/720d9091c8e571d6c891426983f54d9c94739182/docs/write-a-plugin.md#L82
- https://github.com/docsifyjs/docsify/blob/720d9091c8e571d6c891426983f54d9c94739182/docs/index.html#L173
- https://github.com/docsifyjs/docsify/blob/720d9091c8e571d6c891426983f54d9c94739182/index.html#L77

I'd be happy to submit a PR fixing those, if that list is exhaustive, and hardcoding `develop` is the right solution (it would be nice if the default branch could be detected dynamically).

**Comment by sy-records** [comment]
Oh, I see, thanks for the feedback @waldyrious

**Comment by trusktr** [comment]
> I'd be happy to submit a PR fixing those, if that list is exhaustive, and hardcoding `develop` is the right solution (it would be nice if the default branch could be detected dynamically).

@waldyrious That would be great indeed to update those. Not sure about the dynamic part. Maybe GitHub had an API to fetch the default branch?

**Comment by sy-records** [comment]
@waldyrious Can you resubmit a PR for #1458?

**Comment by waldyrious** [comment]
@trusktr thanks for the information! I took a look at the code but it's a little more complex than I anticipated at first.

I suppose one would have to use the [get](https://github.com/docsifyjs/docsify/blob/df21153ab5e841ad89707e07c68a04873a2f3fe8/src/core/fetch/ajax.js#L14) function from `ajax.js` to make the [API request](https://api.github.com/repos/docsifyjs/docsify), then use `JSON.parse` to extract the `default_branch` key from the resulting object; however, the [various replacements](https://github.com/docsifyjs/docsify/blob/720d9091c8e571d6c891426983f54d9c94739182/index.html#L68-L75) done while preparing the edit link seem to require some extra logic to detect the branch from which the site is served. I think I'll leave this to someone more familiar with the code :)

> @waldyrious Can you resubmit a PR for #1458?

@sy-records sure! I just submitted #1466.

File: docs/quickstart.md (hunk 1 above)
Comment (trusktr) [reply docs/quickstart.md:100]: @sy-records let's get things moving. Let's make changes more organic (as per the other v5 topic I closed), and we can release a major version bump any time we fix a small feature that is breaking. In this case, this one is worth it in my opinion.

**Pull request merged**